bundles/model/org.eclipse.smarthome.model.script/META-INF/MANIFEST.MF

```diff
@@ -38,6 +38,7 @@ Import-Package:
  org.eclipse.jdt.annotation;resolution:=optional,
  org.eclipse.smarthome.core.audio,
  org.eclipse.smarthome.core.common.registry,
+ org.eclipse.smarthome.core.ephemeris,

```

Comment:
Is this done by intention that the ephermis bundle is a mandatory requirement for the script model?

Reply:
Well, it seems needed as soon a we want to publish ephemeris methods in rules and scripts. Am I wrong ?